go/vt/discovery/healthcheck.go

```diff
@@ -647,16 +648,24 @@ func (hc *HealthCheckImpl) Unsubscribe(c chan *TabletHealth) {
 	delete(hc.subscribers, c)
 }
 
+var printStack = sync.OnceFunc(func() {
+	buf := make([]byte, 10240) // Allocate buffer large enough
+	n := runtime.Stack(buf, true)
+	fmt.Printf("All Goroutines Stack Trace:\n%s\n", buf[:n])
+})
```

Comment:
Shouldn't this happen only for debugging?

Reply:
Could you please help me understand why it is needed to print all go routines stack trace? 
What information will use be used from it?

Reply:
We can see why the subscriber is blocked and not processing the updates, because we'll be able to see their stack trace.

Reply:
If this is the reason, we can use the pprof to output the logs with
```
pprof.Lookup("goroutine").WriteTo(os.Stdout, 1)
```
runtime.Stack(buf, true) with `true` could do a stop the world to inspect all goroutines

Reply:
Fixed